docs/maintainers/adr/0020-reduce-support-for-internet-explorer.md

```diff
@@ -0,0 +1,77 @@
+# Reduce support for Internet Explorer
+
+* Status: accepted
```

Comment:
Did we say this was going to be `pending` for now? I don't have strong feelings as we're pretty definitely going to do it, just thinking back to the discussions on this as the work as still pending.

Reply:
I thought we'd accepted that we'd have some support, but left enough wiggle room to not specify exactly what at this stage.